stdlib/3/inspect.pyi

```diff
@@ -21,13 +21,21 @@ class BlockFinder:
     def tokeneater(self, type: int, token: str, srow_scol: Tuple[int, int],
                    erow_ecol: Tuple[int, int], line: str) -> None: ...
 
-CO_GENERATOR = ...  # type: int
-CO_NESTED = ...  # type: int
-CO_NEWLOCALS = ...  # type: int
-CO_NOFREE = ...  # type: int
 CO_OPTIMIZED = ...  # type: int
+CO_NEWLOCALS = ...  # type: int
 CO_VARARGS = ...  # type: int
 CO_VARKEYWORDS = ...  # type: int
+CO_NESTED = ...  # type: int
+CO_GENERATOR = ...  # type: int
+CO_NOFREE = ...  # type: int
+
+# Python 3.5+
```

Comment:
These should also use version checks.

Also, we prefer variable annotations (`CO_ASYNC_GENERATOR: int`) for new code.

Reply:
If you would like to, go ahead. It's not required though.

Reply:
I added version checks and variable annotations for all the `inspect.pyi` file

Reply:
On Travis CI for Python `2.7`, there are a lot of `pytype: error: unrecognized arguments`

Reply:
That seems like an issue with pytype rather than your code.

@rchen152 could you take a look?